### ISSUE TITLE
Thread's current page not showing in page title

### ISSUE THREAD

**Issue by Haldeyr** [TuxedoTako/4chan-xt#114]
I've had it working before on my pc and on newest version as well, but not on my laptop. Recently chrome decided to wipe out my scripts, and upon reinstalling XT I can't seem to find the option to enable it?
![image](https://github.com/user-attachments/assets/991ca2df-acc3-4ff7-8fd6-542e2324e39b)

**Comment by TuxedoTako** [comment]
Weird, I can't find it either. ThreadStats.js doesn't modify the title, and Unread.js reads the original title once, and then overwrites it completely, implying that there's nothing else in 4chan X(T) that modifies the title. Except Banner.js, which is the only thing that writes to `Unread.title`.

**Comment by Haldeyr** [comment]
So was it a different script or an edit I got from somewhere else that had it? I remember having to edit 4chan X with picrel to get the purge position to show but that comes by default in XT. I'm so confused ><
![image](https://github.com/user-attachments/assets/595fe903-29fd-4a25-9982-680f303cf88b)

**Comment by TuxedoTako** [comment]
Purge position can be enabled in the settings, but that's in the header on the page, not on the tab title. See #39

**Comment by Haldeyr** [comment]
Yeah I know about that. My issue however is different, my tap would usually look like (0 I 7) indicating that there are 0 unread posts and the thread is at page 7 but that's sadly no longer the case

**Comment by TuxedoTako** [comment]
I checked the git history of Unread.js and ThreadStats.js and I can't find any evidence that this functionality was ever in XT. Were you using a different fork?

**Comment by Haldeyr** [comment]
I don't think so. I've made 2 previous bug reports that got fixed by you, but it might just have been another extra script that's been lost sadly. I think it'd make a nice feature either way, thank you for checking it out ^^
